gcloud/datastore/demo/demo.py

```diff
@@ -11,86 +11,110 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-# pragma NO COVER
 # Welcome to the gCloud Datastore Demo! (hit enter)
-
 # We're going to walk through some of the basics...
 # Don't worry though. You don't need to do anything, just keep hitting enter...
 
-# Let's start by importing the demo module and getting a dataset:
+# Let's start by importing the demo module and initializing our connection.
 from gcloud.datastore import demo
-dataset = demo.get_dataset()
+demo.initialize()
 
 # Let's create a new entity of type "Thing" and name it 'Toy':
-toy = dataset.entity('Thing')
+from gcloud.datastore.key import Key
+key = Key('Thing')
+from gcloud.datastore.entity import Entity
+toy = Entity(key)
```

Comment:
I had planned to operate directly on the keys:  they are much more central to working with the API than they used to be (especially deletion).